back_office/products/migrations/0001_initial.py

```diff
@@ -41,44 +45,45 @@ class Migration(migrations.Migration):
             name='Product',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('title', models.CharField(max_length=255)),
+                ('name', models.CharField(max_length=255)),
+                ('price', models.FloatField(validators=[django.core.validators.MinValueValidator(0.0)])),
```

Comment:
with this validator you can not operate admin panel properly. keeps crashing.
Test your program before pushing it up for Code review